extensions/amp-experiment/0.1/amp-experiment.js

```diff
@@ -89,6 +100,27 @@ export class AmpExperiment extends AMP.BaseElement {
       return experiments;
     });
   }
+
+  /**
+   * Parses URL fragment and returns variant override.
+   * @return {!Object<string, string>}
+   * @private
+   */
+  getVariantOverride_() {
+    const queries = parseQueryString(
```

Comment:
The code saved the hash to location.originalHash. Is it safe to rely on it? is any where else manipulating it?

I did a simple test: When AMP page is inside a viewer, the fragment is available via location.originalHash, when AMP page is on its own, the fragment is available in location.hash.